VaultOfTheIncarnates/Locales/koKR.lua

```diff
@@ -1,65 +1,65 @@
 local L = BigWigs:NewBossLocale("Eranog", "koKR")
 if not L then return end
 if L then
-	-- L.custom_on_nameplate_fixate = "Fixate Nameplate Icon"
-	-- L.custom_on_nameplate_fixate_desc = "Show an icon on the nameplate of Frenzied Tarasek that are fixed on you.\n\nRequires the use of Enemy Nameplates and a supported nameplate addon (KuiNameplates, Plater)."
-
-	-- L.molten_cleave = "Frontal"
-	-- L.incinerating_roar = "Roar"
-	-- L.molten_spikes = "Spikes"
-	-- L.collapsing_army = "Army"
-	-- L.greater_flamerift = "Mythic Add"
-	-- L.leaping_flames = "Flames"
+	 L.custom_on_nameplate_fixate = "시선 고정 네임플레이트 아이콘"
+	 L.custom_on_nameplate_fixate_desc = "당신에게 시선이 고정된 타라세크의 네임플레이트에 아이콘을 표시합니다.\n\n 네임플레이트를 지원하는 애드온(KuiNameplates, Plater)을 사용하고 적 네임플레이트가 활성화된 상태여야함."
+
+	 L.molten_cleave = "전방기"
+	 L.incinerating_roar = "포효"
+	 L.molten_spikes = "쐐기"
+	 L.collapsing_army = "쫄 소환"
+	 L.greater_flamerift = "신화 쫄"
+	 L.leaping_flames = "Flames"
```

Comment:
was this one missed?

Reply:
Here is the ability: https://www.wowhead.com/spell=394917/leaping-flames

Reply:
translated, but I'll definitely keep an eye out on- I think I can definitely find a better word for it